test/extended/deployments/deployments.go

```diff
@@ -1076,8 +1076,9 @@ var _ = g.Describe("[Feature:DeploymentConfig] deploymentconfigs", func() {
 			// Deployment status can't be updated yet but should be right after
 			o.Expect(appsinternalutil.DeploymentStatusFor(rc1)).To(o.Equal(appsapi.DeploymentStatusRunning))
 			// It should finish right after
-			rc1, err = waitForRCModification(oc, namespace, rc1.Name, deploymentChangeTimeout,
+			rc1, err = waitForRCModification(oc, namespace, rc1.Name, deploymentRunTimeout,
```

Comment:
I don't think this was wise. 30s should have been more than enough; even 5s should be more than enough to exit the deployer pod and change a phase on RC. Changing 30s to 5m in timing based test is effectively disabling it - there should have been an issue to track the root cause if this has been disabled to unblock the queue.

Also this looks like broken CRI-O, not the test.

```
2018-07-13T17:56:03.794965111Z --> Scaling minreadytest-1 to 2
2018-07-13T17:57:12.273273657Z --> Success---
```
Printing `Success` is the last line of code in the deployer pod binary, happened at `17:57:12.27` (=> exited) but the pod is still marked as `Running` 30s later at `Jul 13 17:57:42.860`

/cc @mfojtik  (if you were to see any weird flakes)